src/core/init.ts

```diff
@@ -1,11 +1,11 @@
 // @ts-ignore
-import { default as higlassRegister } from 'higlass-register';
+import higlassRegister from 'higlass-register';
```

Comment:
No need for `as` with default imports: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#syntax